### PR TITLE
doc: Add Withdrawal limit to StarkGate architecture

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

New section on StarkGate withdrawal limit

### PR Preview URL

[Withdrawal limit](https://starknet-io.github.io/starknet-docs/pr-1140/documentation/tools/starkgate_architecture/#stark_gate_withdrawal_limit)

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

### CODE REVIEW

File: components/Starknet/modules/tools/pages/starkgate_architecture.adoc [hunk 1]
```diff
@@ -38,6 +38,17 @@ The _StarkGate Registry_::
 * contains the addresses to all supported bridges
 * enables a bridge developer to stop servicing an existing bridge
 
+[#stark_gate_withdrawal_limit]
+== Withdrawal limit
+
+By default, StarkGate imposes no limit on withdrawals. However, in order tBy default, StarkGate imposes no limit on withdrawals. However, in order to mitigate risks associated with critical vulnerabilities that could result in the loss of user funds, StarkGate can enable a withdrawal limit.
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate_architecture.adoc:44]:
```suggestion
By default, StarkGate imposes no limit on withdrawals. However, in order to mitigate risks associated with critical vulnerabilities that could result in the loss of user funds, StarkGate can enable a withdrawal limit.
```

File: components/Starknet/modules/tools/pages/starkgate_architecture.adoc [hunk 2]
```diff
@@ -38,6 +38,17 @@ The _StarkGate Registry_::
...
+
+If a serious security issue arises, the security agent in the StarkGate contract can limit withdrawals to 5% of the Total Value Locked (TVL) per day for any affected token by calling the `enableWithdrawalLimit()` function in the StarkGate contract. A dedicated team can then investigate and resolve the issue.
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate_architecture.adoc:46]: @natan-granit I need a link to the function in the contract.

Comment (natan-granit) [reply components/Starknet/modules/tools/pages/starkgate_architecture.adoc:46]: https://github.com/starknet-io/starkgate-contracts/blob/cairo-1/src/solidity/WithdrawalLimit.sol

File: package-lock.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "starknet-docs",
-  "version": "0.1.483",
+  "version": "0.1.482",
```
Comment (stoobie) [inline package-lock.json:3]:
```suggestion
  "version": "0.1.483",
```

**Review by natan-granit — COMMENTED** [review]
LGTM

**Review by natan-granit — APPROVED** [review]
LGTM

**Pull request merged**